### ISSUE TITLE
The way of converting one page of pdf files to bitmap. 

### ISSUE THREAD

**Issue by ghost** [DImuthuUpe/AndroidPdfViewer#579]
That i use" Steam " source for rendering pdf view in popwindow. and then i want to get a bitmap for one page that i selected of this pdf....there's some way for me? Thanks .
pdfView.fromStream(data)
                .enableAntialiasing(true)
                .defaultPage(0)
                .enableDoubletap(true)
                .swipeHorizontal(true)
                .pageFitPolicy(FitPolicy.BOTH)
                .load();

**Comment by 1stmetro** [comment]
Yes take a bitmap from the view and save the image...

**Comment by andywangzhen** [comment]
Yeah, i want to know the way of that how to get a bitmap from the pdf view .

**Comment by SlinkyRaptor** [comment]
Have a look at PDFFiles renderPageBitmap method or leverage Pdfium directly.

**Comment by huchenzhang** [comment]
ParcelFileDescriptor fd = ParcelFileDescriptor.open(file, ParcelFileDescriptor.MODE_READ_ONLY);
            PdfiumCore pdfiumCore  = new PdfiumCore(this);
            PdfDocument pdfDocument = pdfiumCore.newDocument(fd);

            pdfiumCore.openPage(pdfDocument, 0);
            int width = pdfiumCore.getPageWidthPoint(pdfDocument, 0);
            int height = pdfiumCore.getPageHeightPoint(pdfDocument, 0);

            // ARGB_8888 - best quality, high memory usage, higher possibility of OutOfMemoryError
            // RGB_565 - little worse quality, twice less memory usage
            Bitmap bitmap = Bitmap.createBitmap(width, height,
                    Bitmap.Config.RGB_565);
            pdfiumCore.renderPageBitmap(pdfDocument, bitmap, 0, 0, 0,
                    width, height);
            imageView.setImageBitmap(bitmap);
            pdfiumCore.closeDocument(pdfDocument); // important!

**Comment by kartikj07** [comment]
Hey, @huchenzhang how to create bitmaps for a 50+ page pdf?
